src/app/services/api/api.service.ts

```diff
@@ -81,14 +137,18 @@ export class ApiService {
    * @param endNumber Number (height) of the last block (inclusive).
    */
   getBlocks(startNumber: number, endNumber: number): Observable<GetBlocksResponse> {
-    return this.get('blocks', { start: startNumber, end: endNumber });
+    const url = !this.nodeUrl() ? 'blocks' : 'v1/blocks';
+
+    return this.get(url, { start: startNumber, end: endNumber });
   }
 
   /**
    * Gets information about the state of the blockchain.
    */
   getBlockchainMetadata(): Observable<Blockchain> {
-    return this.get('blockchain/metadata').pipe(
+    const url = !this.nodeUrl() ? 'blockchain/metadata' : 'v1/blockchain/metadata';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    const url = !this.nodeUrl() ? "blockchain/metadata" : 'v1/blockchain/metadata';
```